lib/routes/home/widgets/payments_list/payment_item.dart

```diff
@@ -46,29 +60,33 @@ class PaymentItem extends StatelessWidget {
                               blurRadius: 5.0,
                             ),
                           ],
-                        )
-                      : null,
-                  child: _createdWithin(const Duration(seconds: 10))
-                      ? PaymentItemAvatar(_paymentInfo, radius: 16)
-                      : FlipTransition(
+                        ),
+                  child: isPaymentItemNew
```

Comment:
@erdemyerebasmaz  Does this mean we will get transition for every payment that was created in the last 5 minutes?

Reply:
Yeah, that was pushed by mistake after rebasing. Reduced to 15 seconds.
Added +5 seconds to factor in the delay I mentioned.